packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -764,13 +895,36 @@ export class StatsAnalyzer extends EventsScope {
               `StatsAnalyzer:index#compareLastStatsResult --> No share frames received`,
               currentFramesReceived
             );
+            if (this.lastEmittedStartStopEvent.share.remote === EVENTS.REMOTE_MEDIA_STARTED) {
```

Comment:
for share this will never work, because SDK currently doesn't call emitStartStopEvents() for share, also existing SDK code is incorrectly checking for `this.meetingMediaStatus.expected.sendShare` when checking receive share stats. To make this work properly for share, it requires more work to set `this.meetingMediaStatus.expected.sendShare` and `this.meetingMediaStatus.expected.receiveShare` correctly

Reply:
We can create a JIRA to fix this. In this PR we are only aiming to add all the necessary events.

Reply:
what is the link to that Jira? Also I don't think we should be adding new API events that don't work correctly from the start. If you are actually planning to go ahead with this, then at the very least this should be documented somewhere in the readme or next to the event definitions that these events don't work for share and a jira should be referenced.